src/server/express.ts

```diff
@@ -53,6 +54,7 @@ export async function startExpressServer({
   app.post(path, ...middleware, createMcpServer);
   app.get(path, ...middleware, methodNotAllowed);
   app.delete(path, ...middleware, methodNotAllowed);
+  app.post('/jwt', generateJwt);
```

Comment:
This new endpoint is just for demonstration purposes. Will remove before merging.